apps/console/src/features/claims/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -316,7 +317,7 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
                         />
                     }
                     {
-                        claim &&
+                        claim && claim.displayName !== ClaimManagementConstants.USER_ID &&
```

Comment:
Better if you can wrap the multiline markup with `()` for better formatting.
```suggestion
                        claim && claim.displayName !== ClaimManagementConstants.USER_ID && (
                             <Markup />
                        )
```